src/components/DataTable/index.js

```diff
@@ -52,10 +63,25 @@ export function DataTable ({
 		}, 500);
 	}, [filterValue]);
 
+	// rounding page number divisible by 5
+	function roundedNumRows(x) {
+		if (isNaN(x) == false) {
+			const rowCount = Math.round(x / 5) * 5;
+			if (rowCount <= 0) {
+				return 5;
```

Comment:
@Aringoaway Add a comment here that it is a lowest limit value

Reply:
> @Aringoaway Alright, it looks cool! Only minor stuff to refactor. Has it been also tested when navigating to the screen with datatable from different part of the application?

@Pe5h4 I have tested this on other tables and everything works